components/scream/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ else()
   set (SCREAM_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
 endif()
 
-if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12.0")
+if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.16.0")
```

Comment:
@jgfouca , could this line be causing the `mappy` failures?

Reply:
@AaronDonahue , I bet it is. Let me add a commit to this PR.